src/app/pages/home/home.page.ts

```diff
@@ -100,37 +100,24 @@ export class HomePage implements OnInit {
   }
 
   private getCaptures$() {
-    const proofsWithThumbnail$ = this.proofRepository.getAll$().pipe(
-      concatMap(proofs =>
-        Promise.all(
-          proofs.map(async proof => ({
-            proof,
-            thumbnailBase64: await proof.getThumbnailBase64(),
-          }))
-        )
-      )
-    );
-
     return combineLatest([
```

Comment:
I've found that this observable will not emit value if following the steps:

1. On browser, clean all previous data
2. Login with a clean account
3. Disable network and take 1 capture
4. Even if reload the page, it still won't emit

I did a bit of console log tracking and finds something weird. It's the `proofRepository.getAll$()` that didn't emit value. It seems in capacitor-filesystem-table, the `initialize()` never returns.

Reply:
It seems this won't happen if I enable the network again. And if I disable the network, the problem reappears.